src/lib/layout/shell.svelte

```diff
@@ -92,4 +92,8 @@
             grid-template-columns: auto 1fr !important;
         }
     }
+
+    .main-content {
+        flex-grow: 1;
+    }
```

Comment:
@TGlide I don't think we need this... Elad should have fixed it and it's merged in main

Reply:
![image](https://user-images.githubusercontent.com/26071571/234549366-47a5e45a-715c-45e5-bab7-e51d3d225264.png)
Still happens after merging with main